search/search-client-query-transformer/src/main/java/io/camunda/search/clients/query/SearchQueryBuilders.java

```diff
@@ -264,6 +273,50 @@ public static <C extends Collection<Integer>> SearchQuery intTerms(
     }
   }
 
+  public static List<SearchQuery> intTerms(
```

Comment:
🔧 We can probably rather call those `integerOperations`, `longOperations`, and so on. If I am not mistaken, the word "term" is used for equality checks.

Reply:
Yes, I agree.